docs/docs/user-guides/reverse-tunnel.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Node to Node Connections (Reverse Tunnel)
```

Comment:
I think `Node to Node` implies Peer to Peer (direct connections), so perhaps just `Reverse Tunnel` is a better title?